src/main/java/duke/model/Task.java

```diff
@@ -0,0 +1,56 @@
+package duke.model;
+
+/**
+ * Represents a task to be stored. This object can store the name of the task as a string and the completion status
+ * of the task as a boolean. A true boolean result indicates the task is completed, while a false boolean result
+ * indicates an incomplete task.
+ */
+public abstract class Task {
+    private final String DONE_SYMBOL = "[✓]";
+    private final String UNDONE_SYMBOL = "[✗]";
+    private String taskName;
+    private boolean taskDone;
+
+    //Constructor
```

Comment:
This might be an unnecessary comment as it is clear that the method is the constructor for the class